.envs-rabbitmq

```diff
@@ -0,0 +1,3 @@
+# envs-rabbitmq
+RABBITMQ_USERNAME=georchestra
+RABBITMQ_PASSWORD=georchestra
```

Comment:
Shouldn't we have a new secret for RABBITMQ_PASSWORD ?

Reply:
I don't get it, `envs` files are technically "secrets" because there is no concept of secrets inside docker.

Reply:
I was refering to https://github.com/georchestra/docker/blob/6d084eb9670f54cc738d63489c9f2915eaf27524/docker-compose.yml#L18-L22

Reply:
oh ok, for that to work the container/application has to support the ability to read the password **from a file** (**not from an environment variable**).

I did look at the documentation of the container that wasn't explained anywhere BUT I did an extensive search and found this (for a related container maintained by bitnami): https://github.com/bitnami/containers/issues/46152

We mean yes we can use RABBITMQ_PASSWORD_FILE to password located in file.

Reply:
Excellent, please go ahead.

Reply:
Sound fair to me. Thanks !